### PR TITLE
Plugins Docs

### PR DESCRIPTION
**Pull request by rafaelfragosom**
A plugins section where contributors and maintainers can list their tsuru plugins.

Refs.: #1679

### CODE REVIEW

**Comment by rafaelfragosom** [comment]
@ArxdSilva Let me know if that's okay.

File: docs/plugins/index.rst [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.. Copyright 2012 tsuru authors. All rights reserved.
```
Comment (andrestc) [inline docs/plugins/index.rst]: This should be 2017

File: docs/plugins/index.rst [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+:title: Plugin List
+:description: list of tsuru plugins
+
+.. _plugins:
+
+Plugins
+=======
+
+A list of tsuru plugins.
```
Comment (andrestc) [inline docs/plugins/index.rst]: Maybe add that this plugins are maintained by the community

**Comment by andrestc** [comment]
I'd suggest adding: https://github.com/emerleite/tsuru-bluegreen


Thanks!

**Comment by rafaelfragosom** [comment]
@andrestc Thank you! It's updated.

File: .gitignore [hunk 3]
```diff
@@ -5,3 +5,4 @@ docs/_build
 build
 data
 dist/
+.DS_Store
```
Comment (ArxdSilva) [inline .gitignore]: This should be added on your global .gitignore

Comment (rafaelfragosom) [reply .gitignore]: I understand, but why not?

Comment (ArxdSilva) [reply .gitignore]: Just because that's a auto generated and not necessary.

**Pull request merged**